addons/website_slides/controllers/main.py

```diff
@@ -1133,22 +1133,36 @@ def slide_tag_search_read(self, fields, domain):
 
     @http.route('/slides/embed/<int:slide_id>', type='http', auth='public', website=True, sitemap=False)
     def slides_embed(self, slide_id, page="1", **kw):
-        # Note : don't use the 'model' in the route (use 'slide_id'), otherwise if public cannot access the embedded
-        # slide, the error will be the website.403 page instead of the one of the website_slides.embed_slide.
-        # Do not forget the rendering here will be displayed in the embedded iframe
+        return self._slide_embed(slide_id, page=page, is_external_embed=False, **kw)
+
+    @http.route('/slides/embed_external/<int:slide_id>', type='http', auth='public', website=True, sitemap=False)
```

Comment:
Stupid question: why not an arguement in the route (internal=1, or external=1) ? That way we could keep a single route.

Reply:
Well ok, both are fine to me.